examples/import-import-renaming.kl

```diff
@@ -0,0 +1,4 @@
+#lang kernel
+(import (rename "bool.kl" (not negb)))
+(import "import-renaming.kl")
+(example (negb (false)))
```

Comment:
which of the two `negb` imports should take precedence? the renamed one or the one from `import-renaming. kl`?

Reply:
Oh, I didn't intend to leave the first in there actually :sweat_smile: copy/paste error